### PR TITLE
Automated Test COMMERCE-6157 | master

### PR DESCRIPTION
**Pull request by juliomoraes-dev**

### CODE REVIEW

**Comment by julioribeiiro** [comment]
https://issues.liferay.com/browse/COMMERCE-6157, cc: @lorenzolapescara

**Comment by kristinonias** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 1]
```diff
@@ -253,12 +253,35 @@ definition {
...
+
+		Click(
+			key_navUnderline = "Members",
+			locator1 = "NavUnderline#NAV_UNDERLINE_LINK");
+
+		CommerceAcceleratorsInitializer.inviteNewMemberWithRoleToAccount(
+			emailAddress = "buyer@liferay.com",
+			role = "Buyer");
+
+		ApplicationsMenu.gotoSite(site = "Minium");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:278]: The test does not include these steps.

**Comment by lorenzolapescara** [comment]
Review started.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceAccelerators.path [hunk 2]
```diff
@@ -103,6 +103,11 @@
 	<td>//div[contains(@class,'price')]//span[contains(@class,'price-value-discount')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PRODUCT_DETAILS_VALUE_INACTIVE_PROMO_PRICE</td>
+	<td>//p[@title='${key_productName}']//following-sibling::p[@class='card-text']//span[@class='price-value price-value-inactive']//following-sibling::span[@class='price-value price-value-promo']</td>
+	<td></td>
+</tr>
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceAccelerators.path:110]: This locator is too complex, please try to find and use a path that identifies just the promo price in a product card. Also, please move this locator to `CommerceFrontStore.path`.
```suggestion
<tr>
	<td>PRODUCT_CARD_PROMO_PRICE</td>
	<td>//p[@title='${key_productName}']//following-sibling::p[@class='card-text']//span[contains(@class,'price-value-promo')]</td>
	<td></td>
</tr>
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 3]
```diff
@@ -253,12 +253,35 @@ definition {
...
+
+		CommerceAcceleratorsInitializer.changeProductsSorting(sortingType = "Name Ascending");
+
+		AssertElementPresent(
+			key_productName = "Brake Pads",
+			locator1 = "CommerceAccelerators#PRODUCT_DETAILS_VALUE_INACTIVE_PROMO_PRICE");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:284]: As for the comment above, please update `CommerceAccelerators#PRODUCT_DETAILS_VALUE_INACTIVE_PROMO_PRICE` locator name.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 4]
```diff
@@ -253,12 +253,35 @@ definition {
...
+			locator1 = "NavUnderline#NAV_UNDERLINE_LINK");
+
+		CommerceAcceleratorsInitializer.inviteNewMemberWithRoleToAccount(
+			emailAddress = "buyer@liferay.com",
+			role = "Buyer");
+
+		ApplicationsMenu.gotoSite(site = "Minium");
+
+		CommerceAcceleratorsInitializer.changeProductsSorting(sortingType = "Name Ascending");
+
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:281]: Add another `AssertElementPresent` that checks for the barred list price on a product card on this line. A new locator that identifies the inactive price on a product card is needed.

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by kristinonias** [comment]
@julioribeiiro 

This pull request has conflicting files. Please fix and resend.

**Comment by julioribeiiro** [comment]
fixed

**Comment by julioribeiiro** [comment]
fixed

**Comment by iovaneale** [comment]
ci:test:commerce-functional

**Comment by nikki-pru** [comment]
ci:test:commerce-functional

**Comment by iovaneale** [comment]
ci:forward

**Pull request closed without merging**